corehq/apps/accounting/static/accounting/ko/accounting.payment_method_handler.js

```diff
@@ -10,16 +13,21 @@ var BillingHandler = function (formId, opts) {
     });
 
     self.isWire = ko.observable(opts.isWire || false);
+    self.wireEmails = ko.observable('');
 
     self.paymentIsComplete = ko.observable(false);
     self.paymentIsNotComplete = ko.computed(function () {
         return ! self.paymentIsComplete();
     });
+    self.paymentProcessing = ko.observable(false);
 
     self.serverErrorMsg = ko.observable();
     self.showServerErrorMsg = ko.computed(function () {
         return !! self.serverErrorMsg();
     });
+    self.submitURL = opts.submitURL;
+
+    self.paymentMethod = ko.observable();
```

Comment:
any reason to not use the `isWire` variable for this? do expect there to be more than `wire` and `cc` choices? your call on which to keep, but one should probably go since i think they're redundant